support reproducible builds

Debian has this goal
https://wiki.debian.org/ReproducibleBuilds/

can you please introduce a build option to avoid the use of this?

```
map=/build/sip-tester-3.5.1=. -fstack-protector-strong -Wformat -Werror=format-security -c -o src/sipp-sipp.o `test -f 'src/sipp.cpp' || echo './'`src/sipp.cpp
src/sipp.cpp:1399:34: warning: macro "__DATE__" might prevent reproducible builds [-Wdate-time]
                        " built " __DATE__ ", " __TIME__);
                                  ^~~~~~~~
src/sipp.cpp:1399:48: warning: macro "__TIME__" might prevent reproducible builds [-Wdate-time]
                        " built " __DATE__ ", " __TIME__);
                                                ^~~~~~~~
```

--- a/src/sipp.cpp
+++ b/src/sipp.cpp
@@ -1382,7 +1382,7 @@ int main(int argc, char *argv[])
                 exit(EXIT_OTHER);
             case SIPP_OPTION_VERSION:
                 printf("\n %s.\n\n",
-                       /* SIPp v1.2.3-TLS-PCAP built YMD, HMS */
+                       /* SIPp v1.2.3-TLS-PCAP */
                        "SIPp " SIPP_VERSION
 #ifdef USE_OPENSSL
                        "-TLS"
@@ -1396,7 +1396,7 @@ int main(int argc, char *argv[])
 #ifdef RTP_STREAM
                        "-RTPSTREAM"
 #endif
-                       " built " __DATE__ ", " __TIME__);
+                       );
 
                 printf
                 (" This program is free software; you can redistribute it and/or\n"

